rpm/spec.in

```diff
@@ -93,6 +93,13 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/%{name}/libdrizzle-redux/visibility.h
 
 %changelog
+* Tue Jun 20 2017 Andreas Bok Andersen <andreas.bok@sociomantic.com> - 6.0.5
+- Bug: Fix errors in rpm/spec.in file (2017-06-20)
```

Comment:
I think it should be "fix" (without capital letter). Otherwise the line below is wrong